### ISSUE TITLE
Fix time zone for event date output

### ISSUE THREAD

**Issue by danieltott** [Virtual-Coffee/virtualcoffee.io#61]
## Issue Context

Currently the `dateForDisplay` isn't setting the correct time zone, so it's defaulting to UTC. For now, let's use EST and maybe in the future we can add a dynamic piece that adjusts to the user's local time zone.

![image](https://user-images.githubusercontent.com/360261/96632245-71251b00-12e5-11eb-9014-710369448be1.png)

## Suggested Solution

The `dateForDisplay` filter is a [custom one we created in `.eleventy.js](https://github.com/Virtual-Coffee/virtualcoffee.io/blob/2ea14f1af9e87c4b523bbc799ff14a76fcf818e1/.eleventy.js#L19-L31):

```
const defaultDateOptions = {
  zone: 'edt',
};
eleventyConfig.addFilter(
  'dateForDisplay',
  (dateString, format = 'fff', opts = {}) => {
    const resolvedOptions = {
      ...defaultDateOptions,
      ...opts,
    };
    return DateTime.fromISO(dateString).toFormat(format, resolvedOptions);
  }
);
```

It's using [Luxon's `toFormat` function](https://moment.github.io/luxon/docs/class/src/datetime.js~DateTime.html#instance-method-toFormat), and I incorrectly set the `zone` param (`edt` is a typo, but even if I had typed it correctly it wouldn't work 🙄 ). Luxon docs [have an entry for setting a time zone](https://moment.github.io/luxon/docs/manual/zones.html#specifying-a-zone), and I think either the "IANA" format (`'America/New_York'`) or the "fixed offset" format (`'UTC+7'`) will work well here.

Just a note - since this is happening at build time, if we _don't_ set a time zone, it will default to the time zone of the computer doing the build. So without a time zone set, when I run `start` or `build` on my machine, it is outputting EST times since I live in Cleveland. But the netlify build server is set to UTC, which is why on the site that's what we're seeing.

## Alternatives Considered

Sometime in the future we can do some dynamic stuff but not now.

**Comment by andreamartz** [comment]
Hi Dan.
I'll tackle this one with Courtney.
Andrea

**Comment by danieltott** [comment]
Fixed in #66
Great work and thank you @andreamartz!!

![](https://media3.giphy.com/media/ar4x1w44umngk/giphy.gif)
